controllers/inventory-controller.js

```diff
@@ -0,0 +1,41 @@
+import initKnex from "knex";
+import configuration from "../knexfile.js";
+const knex = initKnex(configuration);
+
+const index = async (_req, res) => {
+  try {
+    // get data from knex db, table inventories
+    const data = await knex("inventories");
```

Comment:
From the example on the ticket, your data is missing *warehouse_name* & it shouldn't include *warehouse_id*, *created_at* & *updated_at*. 

Otherwise, it looks fantastic!!

Reply:
Good catch! I'll do that